Dockerfile

```
@@ -8,9 +8,13 @@ RUN apt-get update \
         patch pkg-config swig unzip wget xz-utils \
         autoconf autotools-dev automake texinfo dejagnu \
         build-essential prelink autoconf libtool libltdl-dev \
-        gnupg2 libdbus-glib-1-2 sudo \
+        gnupg2 libdbus-glib-1-2 sudo curl \
   && rm -rf /var/lib/apt/lists/*
 
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+RUN source $HOME/.cargo/env
+RUN rustup target add wasm32-unknown-emscripten
```

Let's merge these layers,

```suggestion
RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y &&
    source $HOME/.cargo/env &&
    rustup target add wasm32-unknown-emscripten
```

and it would be good it you could build and push this image to your temporary account, then use it in `.circleci/config.yaml` and make sure that removing `make rust` there works. 
Then once CI passes, revert the changes to `.circleci/config.yaml`. Once merged we will release the new docker image via Github actions.

---

@rth Okay I've done this, but it looks like some of the tests have failed? Not sure if those ones usually pass.